# message type that corresponds to inchworm.py
Header header
int8 id
int8[] ee1_pos
int8[] ee2_pos
int8 ee1_status
int8 ee2_status
int8 ee1_shingle_stat
int8 ee2_shingle_stat
int8 behavior